src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Making your website "cross-origin isolated" using COOP and COEP
+subhead: >
+  Use COOP and COEP to set up a cross-origin isolated environment and enable
+  powerful features like `SharedArrayBuffer`, `performance.measureMemory()` or
+  the JS Self-Profiling API.
+description: >
+  Some web APIs increase the risk of side-channel attacks like Spectre. To
+  mitigate that risk, browsers offer an opt-in-based isolated environment called
+  cross-origin isolated. Use COOP and COEP to set up such an environment and
+  enable powerful features like `SharedArrayBuffer`,
+  `performance.measureMemory()` or the JS Self-Profiling API.
+authors:
+  - agektmr
+date: 2020-04-13
+updated: 2020-04-14
+hero: browsing-context-group.png
+tags:
+  - post
+  - security
+---
+Some web APIs increase the risk of side-channel attacks like Spectre. To 
+mitigate that risk, browsers offer an opt-in-based isolated environment called 
+cross-origin isolated. With a cross-origin isolated state, the webpage will be 
+able to use privileged features including:
+
+* [`SharedArrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
```

Comment:
```suggestion
* [`SharedArrayBuffer`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
```

Reply:
This might not be the best thing to do. I'm not sure how up to date or reliable the localized versions of MDN pages are.